core/App/screens/Scan.tsx

```diff
@@ -54,17 +58,58 @@ const Scan: React.FC<ScanProps> = ({ navigation }) => {
     }
   }
 
-  const didDismissCameraDisclosure = () => {
-    dispatch({ type: DispatchAction.DID_SHOW_CAMERA_DISCLOSURE, payload: [] })
+  const permissionFlow = async (method: PermissionContract, permission: Permission) => {
```

Comment:
You have a return type below in `requestCameraUse`, best to add one here for consistency.

Reply:
Will do